.github/workflows/phpstan.yml

```diff
@@ -1,47 +1,24 @@
-name: "PHPStan Static Code Analyses"
+name: PHPStan Static Code Analyses
 
 on:
-  # Run on all pushes and on all pull requests.
-  # Prevent the build from running when there are only irrelevant changes.
-  push:
-    paths:
-      - '**.php'
```

Comment:
Isn't this going to make them run in every PR again, even with irrelevant changes?

Reply:
Yes. It is not possible to seth path to workflows that are called from another workflow. Working on this.

Reply:
Here we go ... https://github.com/sreichel/magento-lts/pull/26